src/Education/Commands/RuleDescriptionUserControl.xaml

```diff
@@ -0,0 +1,10 @@
+﻿<UserControl x:Class="SonarLint.VisualStudio.Education.Commands.RuleDescriptionUserControl"
```

Comment:
This is a bare xaml except for the references which I've copied from the issue viz ones.